Library/Formula/readline.rb

```diff
@@ -33,7 +33,7 @@ class Readline < Formula
 
   def install
     ENV.universal_binary
-    system "./configure", "--prefix=#{prefix}", "--enable-multibyte"
+    system "./configure", "--prefix=#{prefix}", "--enable-multibyte", ("--with-curses" if OS.linux?)
     system "make", "install"
```

Comment:
The error message itself was rather cryptic and occured while configure'ing cgdb. Something along the lines of "unable to check version of readline". Actually, in config.log there was more precise message that showed missing ncurses symbols during linking stage.

Readline itself builds and installs fine even without this patch but is simply unusable.